### DISCUSSION TITLE
虽然被晒黑了一点，但至少没有分色

### DISCUSSION THREAD

**Discussion by jsishsjsj** [share121/inter-knot#4794] [灌水]
![Image_1726068814693](https://github.com/user-attachments/assets/f248a3f4-e4cc-42b5-865a-65aa5163bad7)
![120263853(32274677)_もこもこ猫ホシノ_0](https://github.com/user-attachments/assets/39b0c1c1-24f8-4570-a4fe-88f58521061a)

**Comment by Junjun280** [comment]
好看爱看<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by The-Place-Where-Winter-Sieeps** [comment]
好色😍😍😍

**Reply by jsishsjsj** [reply]
？？？

**Comment by tingyu919** [comment]
兄弟你香香的

**Comment by AcheronMei** [comment]
我是lsp我爱看

**Comment by Kunspring** [comment]
这是我可以看的吗（（

**Comment by lllliufeng** [comment]
造化钟神秀，阴阳割昏晓（雾

**Reply by jsishsjsj** [reply]
？？？

**Comment by Fluorine-z** [comment]
![bd5b2b0e613e91b36b65602bfecf8f3330727087](https://github.com/user-attachments/assets/4f725fa3-d00a-4bb9-bd78-676758805457)
兄弟你好香

**Comment by lnx082** [comment]
兄弟你好香

**Comment by Finkawa** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/01/18/8481df2cd99d2b00ecb94a92a14c9fe4_7915718047845434249.png" alt="米游兔 OK" width="75" >_少年好腰_

**Comment by LiuLiBHZ** [comment]
芥末好？爱看

**Comment by HoshinoHanashi** [comment]
兄弟你好香啊
